pkg/fanal/image/daemon/docker_test.go

```diff
@@ -37,7 +38,7 @@ func TestDockerImage(t *testing.T) {
 			ref, err := name.ParseReference(tt.imageName)
 			require.NoError(t, err)
 
-			_, cleanup, err := DockerImage(ref, "")
+			_, cleanup, err := DockerImage(context.Background(), ref, "")
```

Comment:
You can use `t.Context()`
```suggestion
			_, cleanup, err := DockerImage(t.Context(), ref, "")
```